tests/scm/scm-test-arity.scm

```diff
@@ -0,0 +1,43 @@
+
+(use-modules (opencog) (opencog test-runner))
+(use-modules (opencog python))
+
+(opencog-test-runner)
+(define t "opencog-arity-test")
+
+(test-begin t)
+
+(python-eval "
+from opencog.atomspace import AtomSpace, TruthValue
+from opencog.atomspace import types
+def foo(atspace):
+    TV = TruthValue(0.2, 0.69)
+    atspace.add_node(types.ConceptNode, 'Apple', TV)
+")
+
+(python-call-with-as "foo" (cog-atomspace))
+(cog-node 'ConceptNode "Apple")
+
+; Return the strength of a simple truth value
+(define (get-tv-strength tv) (cdr (assoc 'mean (cog-tv->alist tv))))
```

Comment:
this is harmless, but not needed. The built-in `cog-tv-mean` does the same thing. I do not know why it is sometimes called "mean" and sometimes called "strength".

in case you did not know typing `,a` and `,d` at the guiel prompt brings up documentation `,a` is the same as `,apropos` and `,d` is the same as `describe`. So:
```
 $ guile
Enter `,help' for help.
scheme@(guile-user)> (use-modules (opencog))
scheme@(guile-user)> ,a cog-tv
(opencog): cog-tv-mean	#<procedure cog-tv-mean (_)>
(opencog): cog-tv-merge-hi-conf	#<procedure cog-tv-merge-hi-conf (_ _)>
(opencog): cog-tv->alist	#<procedure cog-tv->alist (_)>
(opencog): cog-tv-count	#<procedure cog-tv-count (_)>
(opencog): cog-tv	#<procedure cog-tv (_)>
(opencog): cog-tv?	#<procedure cog-tv? (_)>
(opencog): cog-tv-confidence	#<procedure cog-tv-confidence (_)>
(opencog): cog-tv-merge	#<procedure cog-tv-merge (_ _)>
scheme@(guile-user)> ,d cog-tv-mean

 cog-tv-mean TV
    Return the `mean` of the TruthValue TV. This is a single
    floating point-number.

scheme@(guile-user)> 
```